src/styles/blocks/timer.scss

```diff
@@ -0,0 +1,39 @@
+.timer {
+  @include flex ($flex, $center);
+  @include decoration ($timer-width, $timer-height, $timer-color, $fifth);
+  position: $relative;
+
+  &__minit {
+    position: $absolute;
+    bottom: $fifth;
+    @include decoration ($minit-width, $minit-height, $minit-color, $arrow-rad);
+
+    animation: move 45s 15s linear infinite;

```

Comment:
Currently your minute hand moves with the same speed as seconds hand, please fix that